Add author avatar to the archive page

laurelfulford opened the pull request:
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds the author avatar, if present, to the author archive page.

See #350.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Confirm the author avatar now appears on the author archive pages, next to the information about them:

![image](https://user-images.githubusercontent.com/177561/66722022-bc951700-edbd-11e9-82af-68d7947178d0.png)

![image](https://user-images.githubusercontent.com/177561/66722026-cb7bc980-edbd-11e9-9fb2-881d0ae38ecf.png)

![image](https://user-images.githubusercontent.com/177561/66722027-d5053180-edbd-11e9-90fb-a94434d16016.png)

3. Check the appearance on different screensizes.
4. Navigate to Settings > Discussion, and hide avatars; confirm this removes them from the author archive page, too.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

thomasguillot commented on archive.php:
```diff
@@ -14,9 +14,33 @@
+									'height' => array(),
+								),
+							)
+						);
```
I don't think you need `wp_kses()` here. A simple `echo $author_avatar` should do the trick.

Also for some reason, with Jetpack (and image lazy loading) enabled, the avatar is being displayed twice: [see screenshot](https://cld.wthms.co/YrxfPw) but it's fine when using `echo $author_avatar`

thomasguillot replied:
Note: The avatar being displayed twice appears everywhere.

https://cld.wthms.co/8QllP6

https://cld.wthms.co/UJRlWr

laurelfulford replied:
Thanks Thomas! This should be fixed in 8f62fd7.

I was getting a bit too clever about avoiding a PHPCS error; echoing just `get_avatar()` works just as well. I'll fix that elsewhere in the site in a separate PR.

laurelfulford commented:
I've fixed the avatar duplication issue, as well as a couple other things:

For some reason the avatar was squished on load in Firefox, then would correct itself when I inspected it. I added a `flex-shrink` value to fix this.

I completely forgot that the archive headers were centred in style 4, so I also added some adjustments there to make the avatar display better.

thomasguillot commented on archive.php:
```diff
@@ -19,18 +19,7 @@
-							)
-						);
+						echo get_avatar( $author_id, 120 );
 					}
```
We can simplify this with: `if ( $author_avatar ) echo $author_avatar;` rather than duplicating `get_avatar( $author_id, 120 );`

laurelfulford replied:
Ah good call - thanks Thomas! This has been updated in 3950ae7 .

laurelfulford commented:
Thanks Thomas!

I meant to add this in my note, but we have a PHPCS check that runs on commit, so I couldn't place the function all on one line or else it gets cranky; I also added a line of code so it would ignore `$author_avatar` (which it threw an error for:

` All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$author_avatar'. (WordPress.Security.EscapeOutput.OutputNotEscaped)`

thomasguillot commented:
> I meant to add this in my note, but we have a PHPCS check that runs on commit, so I couldn't place the function all on one line or else it gets cranky; I also added a line of code so it would ignore `$author_avatar`

Weird 😅 Anyway, this looks great! 😊
